docs/getting-started/create-block/block-anatomy.md

```diff
@@ -33,7 +33,7 @@ The first parameter in the **registerBlockType** function is the block name, thi
 
 The second parameter to the function is the block object. See the [block registration documentation](/docs/reference-guides/block-api/block-registration.md) for full details.
 
-The last two block object properties are **edit** and **save**, these are the key parts of a block. Both properties are functions that are included via the import above.
+The two block object properties are **edit** and **save**, these are the key parts of a block. Both properties are functions that are included via the import above.
```

Comment:
This change is incorrect. There are more than two properties. Instead, I would say something like:

```suggestion
Two common object properties are **edit** and **save**, these are the key parts of a block. Both properties are functions that are included via the import above.
```